src/db.ts

```diff
@@ -302,15 +308,34 @@ export const saveBooking = (accountId, booking) => {
     .then(savePerson);
 };
 
-export const getAllPersons = async (
-  sort: boolean = false
+/**
+ * Finds persons. When callbackFn is not supplied, loads all persons.
+ * Notes:
+ *  Avoid using this function without a callbackFn
+ *  If you need to find one person, you can use findPerson() instead
+ * @param sort
+ * @param callbackFn
+ */
+export const findPersons = async (
```

Comment:
:thinking: I got the error below. (note that `mock-solaris` works, I can login into mobile app)

![Screenshot from 2023-06-07 21-28-11](https://github.com/kontist/mock-solaris/assets/74178206/6a6f5f2c-c6ad-48a9-a735-5d13f83c2770)

Reply:
Related commit: https://github.com/kontist/mock-solaris/pull/114/commits/98e05597df1b9c05dde14832ebc3dfe9ab338f13